airbyte-workers/src/main/java/io/airbyte/workers/DefaultSyncWorker.java

```diff
@@ -82,6 +88,21 @@ public DefaultSyncWorker(final Source<T> source,
 
       destination.notifyEndOfStream();
 
+      final boolean shouldNormalize = Optional
+          .ofNullable(syncInput.getDestinationConnection().getConfiguration().get(WorkerConstants.BASIC_NORMALIZATION_KEY))
+          .map(JsonNode::asBoolean)
+          .orElse(false);
+      if (shouldNormalize) {
+        try (normalizationRunner) {
+          LOGGER.info("Running normalization.");
+          final Path normalizationRoot = Files.createDirectories(jobRoot.resolve("normalize"));
+          if (!normalizationRunner.normalize(normalizationRoot, syncInput.getDestinationConnection().getConfiguration(), syncInput.getCatalog())) {
+            throw new WorkerException("Normalization Failed.");
+          }
+        } finally {
+          normalizationRunner.close();
```

Comment:
Why do you call the close btw since you're using the try-resource?

Reply:
artifact of previous iteration. removed.